src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -38,50 +39,54 @@
 
 namespace ripple {
 
-static std::optional<uint256>
-parseIndex(Json::Value const& params, Json::Value& jvResult)
+static Expected<uint256, Json::Value>
+parseObjectID(
+    Json::Value const& params,
+    Json::StaticString const& fieldName,
```

Comment:
I think everywhere `Json::StaticString const&` should be `Json::StaticString const`, because it's a pointer wrapper, and it's better to pass by value.

Reply:
Fixed in [f380124](https://github.com/XRPLF/rippled/pull/5237/commits/f380124966119c33193e8b91fdb857d521f70953)